docs/events/jedi-2023-annual-meeting.html

```diff
@@ -0,0 +1,137 @@
+---
+title: Data-PASS Journal Editors Discussion Interface (JEDI)
+layout: main
+---
+<head>
+    <style>
```

Comment:
As you do more with styles, at some point you'll want to apply side-wite CSS rules and work with HTML elements in the pages.